### PR TITLE
resource_manager: change cpu time unit from ms to ns

### PR DESCRIPTION
**Pull request by glorv**
### What problem does this PR solve?

Issue Number: Ref #5851

### What is changed and how does it work?

```
Change the cpu time uint from ms to ns because ms is too coarse-grained that it will ignore duration small than 1ms.
```

### Check List

Tests

- Unit test

Code changes

Side effects

- Breaking backward compatibility

Related changes

- PR to update [`pingcap/docs`](https://github.com/pingcap/docs)/[`pingcap/docs-cn`](https://github.com/pingcap/docs-cn):
- PR to update [`pingcap/tiup`](https://github.com/pingcap/tiup):
- Need to cherry-pick to the release branch

### Release note

```release-note
None.
```

### CODE REVIEW

File: client/resource_group/controller/model.go [hunk 1]
```diff
@@ -38,9 +39,9 @@ type RequestInfo interface {
 
 // ResponseInfo is the interface of the response information provider. A response should be
 // able to tell how many bytes it read and KV CPU cost in milliseconds.
-type ResponseInfo interface {
+type RespontokenConsumptionPerSecondseInfo interface {
```
Comment (JmPotato) [inline client/resource_group/controller/model.go]: Why change the naming?

Comment (glorv) [reply client/resource_group/controller/model.go]: fix, this was a accidental typo

**Review by JmPotato — APPROVED** [review]

**Review by nolouch — APPROVED** [review]

**Pull request merged**